sandbox/grommet-app/src/pages/layouts/kinds/HPEMarketing/HPEArchitect.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { useContext } from 'react';
+import {
+  Box,
+  Button,
+  Grid,
+  Heading,
+  Image,
+  PageContent,
+  Text,
+  ResponsiveContext,
+} from 'grommet';
+import { LinkNext } from 'grommet-icons';
+import { Metric } from '../../../../components';
+
+export const HPEArchitect = ({ size }) => {
+  const breakpoint = useContext(ResponsiveContext);
+
+  const columns = ['xsmall', 'small'].includes(breakpoint)
+    ? ['auto']
+    : ['3/4', 'auto'];
+  const gap = {
+    xsmall: 'medium',
+    small: 'medium',
+    medium: 'medium',
+    large: 'large',
+    xlarge: 'large',
+  };
+
+  const metricColumns = ['xsmall', 'small'].includes(breakpoint)
+    ? ['auto']
+    : { count: 3, size: 'auto' };
+
+  const areasDefault = [['quotes', 'hpeBook']];
+
+  const areasSmall = [
+    ['quotes', 'unassigned'],
+    ['hpeBook', 'unassigned'],
+  ];
+
+  const areas = ['xsmall', 'small'].includes(breakpoint)
+    ? areasSmall
+    : areasDefault;
+
+  return (
+    <PageContent
+      pad="large"
+      background={{
+        fill: 'horizontal',
+        // this is the closet color we have to what they have #00384D
+        color: 'graph-1',
+        dark: true,
+      }}
+    >
+      <Grid areas={areas} columns={columns} gap={gap[breakpoint]}>
+        <Box gap="xlarge" gridArea="quotes">
+          {/* // this heading is not large enough can we use size or we steering away? */}
+          <Box gap="medium">
+            <Heading size="xlarge" margin="none" level={2}>
+              Architect an AI advantage
+            </Heading>
+            <Text>
+              Overcoming overconfidence and oversight to help leaders secure
+              long-term business success.
+            </Text>
+          </Box>
+          <Box gap="large">
+            <Grid rows="auto" columns={metricColumns} gap="medium">
+              {/* we do not have a text size for these values that should be in 
```

Comment:
Can you clarify this comment? We do have values bigger than "xxlarge". We have 3xl, 4xl, 5xl

Reply:
your right I forgot about those larger values looks like I can use 4xl